src/cockpit/389-console/src/lib/replication/replConfig.jsx

```diff
@@ -243,18 +260,22 @@ export class ReplConfig extends React.Component {
             passwd: this.state.manager_passwd,
             addNotification: this.props.addNotification,
             msg: _("Replication Manager"),
-            success_msg: _("Successfully added Replication Manager"),
-            error_msg: _("Failure adding Replication Manager"),
+            success_msg: this.state.editManager ?
+                "Successfully edited Replication Manager" :
+                _("Successfully added Replication Manager"),
+            error_msg:this.state.editManager ?
+                "Failure editing Replication Manager" :
```

Comment:
Missing internationalization tags - `_("")`

Reply:
> Missing internationalization tags - `_("")`

We don't have the corresponding PO mapping, so it wouldn't do anything

Reply:
Leaving it out is one way to tell that it would need to be done, but we need someone who knows Japanese to do it